src/Logging/Logging.EventSource/test/EventSourceLoggerTest.cs

```diff
@@ -760,7 +760,7 @@ static string GetEscapedForwardSlash(string input)
             {
                 if (c == '/')
                 {
-                    output += "\\u" + ((int)c).ToString("x4");
+                    output += "\\/";
```

Comment:
We are no longer escaping forward slash by default: https://github.com/dotnet/corefx/pull/39494

Reply:
I think that's what we're reacting to here.

Though @BrennanConroy ... this entire method seems to be a slow no-op now :)

Reply:
@anurse What?

Reply:
Never mind, I see the double escape now.